templates/ui.html

```diff
@@ -2131,9 +2158,45 @@
           }
         },
         mounted() {
-          this.updateDefaultOptions("default", this.initializeFilerobot);
+          this.updateDefaultOptions(
+              "default",
+              () => {this.initializeFilerobot(), this.startIntroJS()}
+          );
         },
       }).use(vuetify).mount('#app');
     </script>
+    <script>
+      function getIntroJS() {
+        return introJs().setOptions({
+          tooltipClass: "text-black",
+          showProgress: true,
+          showBullets: false,
+          steps: [
+            {
+              title: "Refocus Quick Guide",
+              element: document.querySelector("#prompt-textarea"),
+              intro: "Just simply describe what you would like to see in plain English here.",
+            },
+            {
+              element: document.querySelector("#generate-button"),
+              intro: "Click here to generate images.",
+            },
+            {
+              element: document.querySelector("#preset-clip"),
+              intro: "Click here to choose a preset for paramters and styles.",
+            },
+            {
+              element: document.querySelector("#img2img-button"),
+              intro: "Click here to open the image to image interface.",
+            },
+            {
+              element: document.querySelector("#introjs-button"),
+              title: "Enjoy the Refocus",
+              intro: "Click here to view guide again. <p> Join our <a href='https://discord.gg/e4UVBNuHyB'>Discord</a> for futher support.</p><br><p>Enjoy!</p>",
```

Comment:
"Click here to view this guide again. <p> Join our <a href='https://discord.gg/e4UVBNuHyB'>Discord</a> for futher support.</p><br><p>Enjoy!</p>"